### PR TITLE
Update mod to Minecraft 1.19.4

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+minecraft_version=1.19.4-rc2
+yarn_mappings=1.19.4-rc2+build.1
+loader_version=0.14.17
 
 #Dependencies
-fabric_version=0.68.1+1.19.3
+fabric_version=0.75.3+1.19.4
 
 # Mod Properties
 mod_version=0.3.4
```
Comment (Gegy) [inline gradle.properties:12]: Bump version? 🙂

**Review by Gegy — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (haykam821) [reply gradle.properties:12]: Should it be bumped to 0.3.5 or 0.4.0? When updating from Minecraft 1.19.1 to 1.19.3, the version stayed the same.

Comment (Gegy) [reply gradle.properties:12]: Ahh, probably should have bumped then too - but I think 0.3.5 makes sense to me 🙂

Comment (haykam821) [reply gradle.properties]: Fixed in 5aeb45e2b5032ad3dfe59cd52e3ccc011dd95d18

**Review by Gegy — APPROVED** [review]

**Pull request merged**